apps/web/vite.config.ts

```
@@ -1,14 +1,10 @@
+import { nitro } from "nitro-nightly/vite";
 import { defineConfig } from "vite";
 import tsconfigPaths from "vite-tsconfig-paths";
 
 export default defineConfig({
-  plugins: [
-    tsconfigPaths(),
-    tailwindcss(),
-    tanstackStart({ customViteReactPlugin: true, target: "vercel" }),
-    viteReact(),
-  ],
+  plugins: [tsconfigPaths(), tailwindcss(), tanstackStart(), nitro(), , viteReact()],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)